client/packages/invoices/src/Returns/CustomerDetailView/AppBarButtons.tsx

```diff
@@ -37,8 +37,7 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
           onClick={() => onAddItem()}
         />
         <ReportSelector
-          context={ReportContext.InboundShipment}
-          subContext="CustomerReturn"
+          context={ReportContext.InboundReturn}
```

Comment:
Would have this as CustomerReturn, and below as SupplierReturn, so these both need to be added to ContextType in backend... and would probably do a migration to drop InboundReturn and OutboundReturn 👀